mobilecoind/src/payments.rs

```diff
@@ -836,6 +843,7 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
     ///   default one (EmptyMemoBuilder).
     /// * `rng` - randomness
     /// * `logger` - Logger
+    #[allow(clippy::too_many_arguments)]
```

Comment:
:question: Do we have any plan to follow up on this in a subsequent PR/issue. This is the second time I've seen the too_many_arguments allow statement come in, in the past week...

Reply:
I don't think so. This is likely low priority at the moment.